src/backend/common/SerialQueue.h

```diff
@@ -110,12 +110,14 @@ namespace backend {
 
     template<typename T>
     void SerialQueue<T>::Enqueue(const std::vector<T>& values, Serial serial) {
+        ASSERT(values.size() > 0);
```

Comment:
Ah sorry I misread and didn't realize in was for the values that were enqueued. ASSERT sounds good now :)